### ISSUE TITLE
Upgrade JSII automatically

### ISSUE THREAD

**Issue by skorfmann** [hashicorp/terraform-cdk#1206]
In order to see issues with new jsii releases as early as possible, we'll have to have an auto-upgrade workflow which takes care of this.

**Comment by DanielMSchmidt** [comment]
Isn't this already done by Dependabot?

**Comment by DanielMSchmidt** [comment]
No, we first need to unblock that JSII can be upgraded (with the python issues)

**Comment by ansgarm** [comment]
Right – but then it'll be done by Dependabot 👍

**Comment by DanielMSchmidt** [comment]
Lots of snapshots have the JSII version and other info related to a specific JSII version in it. To have successful Dependabot merges we need to remove these parts from the snapshot. We need to make a small change here to how we write the snapshots / what we test.
